assignment/assignment1/server/src/index.ts

```diff
@@ -52,6 +52,7 @@ app.use("/auth", authRoutes);
 app.use("/users", userRoutes);
 app.use("/questions", questionRoutes);
 app.post("/questions", verifyToken, createQuestion);
+app.get("/questions", verifyToken, getAllQuestions);
```

Comment:
I think we have already registered this route in the questionRoutes? Is there a reason why we are declaring it again?

Reply:
Thanks for highlighting 😄